### PR TITLE
Adapt SDK with latest swagger file

### PR DESCRIPTION
**Pull request by Flanker32**
Adapt SDK with latest swagger file

### CODE REVIEW

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java [hunk 1]
```diff
@@ -122,4 +127,13 @@ public AppResourceInner getApp() {
     public String getAppName() {
         return appName;
     }
+
+    private boolean isEnablePersistentStorage(SpringConfiguration configuration) {
```
Comment (andxu) [inline azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java]: static

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java [hunk 2]
```diff
@@ -122,4 +127,13 @@ public AppResourceInner getApp() {
...
+        return configuration != null && configuration.getDeployment() != null && configuration.getDeployment().isEnablePersistentStorage();
```
Comment (andxu) [inline azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java]: is configuration.getDeployment().isEnablePersistentStorage() returns Boolean or boolean?

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java (hunk 1 above)
Comment (Flanker32) [reply azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java]: Fixed

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java (hunk 2 above)
Comment (Flanker32) [reply azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java]: It's a Boolean value generated by `BooleanUtils.isTrue()`

File: azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java [hunk 3]
```diff
@@ -122,4 +127,13 @@ public AppResourceInner getApp() {
     public String getAppName() {
         return appName;
     }
+
+    private static boolean isEnablePersistentStorage(SpringConfiguration configuration) {
+        return configuration != null && configuration.getDeployment() != null && configuration.getDeployment().isEnablePersistentStorage();
+    }
+
+    private PersistentDisk getPersistentDiskOrDefault(AppResourceProperties appResourceProperties) {
```
Comment (andxu) [inline azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java]: static getPersistentDiskOrDefault?

Comment (Flanker32) [reply azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/spring/SpringAppClient.java]: Fixed

**Pull request merged**